isic_archive/tasks/app.py

```diff
@@ -49,26 +52,25 @@ class CeleryAppConfig(object):
     # jsonpickle is used to support passing object ids between tasks
     task_serializer = 'jsonpickle'
 
+    # TODO: Remove when Celery version >= 4.3 since it then infers from env vars
+    result_backend = os.getenv('CELERY_RESULT_BACKEND')
+
 
 app.config_from_object(CeleryAppConfig())
 
 
 @app.on_after_configure.connect
 def setupPeriodicTasks(sender, **kwargs):
-    from isic_archive_tasks.notification import maybeSendIngestionNotifications
+    from isic_archive.tasks import maybeSendIngestionNotifications
     sender.add_periodic_task(30, maybeSendIngestionNotifications.s(),
                              name='Send any necessary notifications for ingested batches.')
 
 
 @worker_process_init.connect
-def fixGirderImports(sender, **kwargs):
-    from girder.utility.server import configureServer
-    configureServer(curConfig={
-        'server': {
-            'mode': 'production'
-        },
-        'cherrypy_server': False
-    })
+def addMailTemplates(sender, **kwargs):
+    mail_utils.addTemplateDirectory(
```

Comment:
Can you add a comment of something like "This is a bare-bones version of `IsicArchive.load`".